Software/Scratch/GoPiGoScratch.py

```diff
@@ -359,32 +359,22 @@ def run(self):
 		elif msg.lower()=="IR".lower():
 			print "IR!"
 			if en_ir_sensor==0:
-				import lirc
-				try:
-					sockid = lirc.init("keyes", blocking = False)
-					en_ir_sensor=1
-				except:
-					en_ir_sensor=0
-					print("*******************************")
-					print("ERROR: IR Receiver not enabled!")
-					print("Please run Advanced Communications Option and enable it")
-					print("A REBOOT will be necessary")
-					print("*******************************")
+				import ir_receiver
+				en_ir_sensor=1
```

Comment:
I don't think there's a good reason for including the extra IR.  I think it should be redundant.